src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -2390,6 +2390,15 @@ public class SimpleTypeWihtMoreProperties
         public List<string> CollectionProperty { get; set; }
         public List<SimpleTypeWihtMoreProperties> SimpleTypeList { get; set; }
     }
+
+    public class TypeWith2DArrayProperty
```

Comment:
In general it'd be better to use one type for one test case. If Net Native tools failed to generate serializer for one type, all related tests would fail.